peer/brontide.go

```diff
@@ -3793,3 +3844,37 @@ func (p *Brontide) handleNewActiveChannel(req *newChannelMsg) {
 	// Close the err chan if everything went fine.
 	close(req.err)
 }
+
+// handleNewActiveChannel handles a `newChannelMsg` request. Depending on we
+// know this channel ID or not, we'll either add it to the `activeChannels` map
+// or init the next revocation for it.
+func (p *Brontide) handleNewPendingChannel(req *newChannelMsg) {
+	defer close(req.err)
+
+	chanID := req.channelID
+
+	// If we already have this channel, something is wrong with the funding
+	// flow as it will only be marked as active after `ChannelReady` is
+	// handled. In this case, we will do nothing but log an error, just in
+	// case this is a legit channel.
+	if p.isActiveChannel(chanID) {
+		p.log.Errorf("Channel(%v) is already active, ignoring "+
+			"pending channel request", chanID)
+
+		return
+	}
```

Comment:
So we won't load these into the link? If not, then we won't immediately start to send chan reest to mark the new session as active/eligible.

Reply:
Updated the docs in `handleNewPendingChannel`.